### DISCUSSION TITLE
Emulating fritzbox firmware inside a qemu environment

### DISCUSSION THREAD

**Discussion by Shinxyy** [Freetz-NG/freetz-ng#968] [Allgemein]
Hey everyone,

I'm currently working on rehosting the firmware for the Fritzbox 7530 router. I'm using a QEMU environment to make vulnerability testing with the fuzzing process easier. Has anyone here ever tried emulating the Fritzbox firmware with QEMU or another emulator? Any help would be greatly appreciated.

Thanks a lot!

Für diejenigen, die kein Englisch sprechen:
Hallo zusammen,

Ich hoffe, es geht euch allen gut. Ich wollte mich melden, weil ich im Prozess des Neuaufsetzens der Firmware für den Fritzbox 7530 Router auf eine kleine Hürde gestoßen bin. Trotz meiner Bemühungen mit der QEMU-Umgebung für die Schwachstellentests scheine ich an einem bestimmten Punkt festzustecken.

Hat jemand ähnliche Probleme erlebt oder vielleicht Ratschläge, wie ich weitermachen soll? Jegliche Einblicke oder Hilfe wären äußerst hilfreich.

Vielen Dank für eure Zeit und Unterstützung!

**Comment by fda77** [comment]
Das wurde früher mal gemacht, aber es fehlten Funktionen wie dect usw, AVM steht auf closed source Krams. Das würde dir aber auch nicht helfen da es ARM Fritzboxen wie deine noch nicht so lange gibt. Daher: ein
